src/app/Library/CrudPanel/Traits/SaveActions.php

```diff
@@ -417,4 +417,26 @@ public function setupDefaultSaveActions()
 
         $this->addSaveActions($defaultSaveActions);
     }
+
+    /**
+     * Here we check if user should be warned before leaving the page.
+     *
+     * @return bool
+     */
+    public function getWarnBeforeLeave(): bool
+    {
+        return $this->getOperationSetting('warnBeforeLeave') ?? config('backpack.crud.operations.'.$this->getCurrentOperation().'.warnBeforeLeave') ?? false;
+    }
+
+    /**
+     * Change the variable that determines if user should be warned before leaving the page.
+     *
+     * @param bool $value
+     *
+     * @return void
+     */
+    public function setWarnBeforeLeave(bool $value = true)
+    {
+        $this->setOperationSetting('warnBeforeLeave', $value);
+    }
```

Comment:
Well done @jnoordsij ! I think you've matched our coding style very well.

I would move this to a different trait, though - in my opinion this feature doesn't have much to do with the SaveActions - it's more something related to the Create/Update operation. But it could also be used on any other operation with a form (I guess) so I think it'd be better if we separated this into its own trait, say `WarnBeforeLeaving`.